modules/terraform/clients.nix

```diff
@@ -316,7 +326,7 @@ in {
         lib.nameValuePair vpc.region {
           provider = awsProviderFor vpc.region;
           vpc_peering_connection_id =
-            id "aws_vpc_peering_connection.${vpc.region}";
+            id "aws_vpc_peering_connection_accepter.${vpc.region}";
```

Comment:
I intuitively would understand that changing the options wouldn't recreate any peer network resources, but it might be good to have confirmation on that.

Looking at the examples in the docs, the actual peering connections doesn't seem to have a dependency on these options: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_peering_connection_options

Reply:
lol none of this was intuitive 😅